app/views/reviews/_review.html.haml

```diff
@@ -0,0 +1,9 @@
+%div.review-container
+  %div.voting-title-container
```

Comment:
`%div.voting-title-container` can be written as `.voting-title-container` since `%div` is implicit